From: SeongJae Park <sj@kernel.org>
Subject: mm/damon/core: remove 'struct target *' parameter from damon_aggregated tracepoint
Date: Thu, 7 Sep 2023 02:29:29 +0000

damon_aggregateed tracepoint is receiving 'struct target *', but doesn't
use it.  Remove it from the prototype.

Link: https://lkml.kernel.org/r/20230907022929.91361-12-sj@kernel.org
Signed-off-by: SeongJae Park <sj@kernel.org>
Cc: Jonathan Corbet <corbet@lwn.net>
Cc: Steven Rostedt (Google) <rostedt@goodmis.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/trace/events/damon.h |    6 +++---
 mm/damon/core.c              |    2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/include/trace/events/damon.h~mm-damon-core-remove-struct-target-parameter-from-damon_aggregated-tracepoint
+++ a/include/trace/events/damon.h
@@ -11,10 +11,10 @@
 
 TRACE_EVENT(damon_aggregated,
 
-	TP_PROTO(struct damon_target *t, unsigned int target_id,
-		struct damon_region *r, unsigned int nr_regions),
+	TP_PROTO(unsigned int target_id, struct damon_region *r,
+		unsigned int nr_regions),
 
-	TP_ARGS(t, target_id, r, nr_regions),
+	TP_ARGS(target_id, r, nr_regions),
 
 	TP_STRUCT__entry(
 		__field(unsigned long, target_id)
--- a/mm/damon/core.c~mm-damon-core-remove-struct-target-parameter-from-damon_aggregated-tracepoint
+++ a/mm/damon/core.c
@@ -776,7 +776,7 @@ static void kdamond_reset_aggregated(str
 		struct damon_region *r;
 
 		damon_for_each_region(r, t) {
-			trace_damon_aggregated(t, ti, r, damon_nr_regions(t));
+			trace_damon_aggregated(ti, r, damon_nr_regions(t));
 			r->last_nr_accesses = r->nr_accesses;
 			r->nr_accesses = 0;
 		}
_
